Common/Domain/Domain.fsproj

```diff
@@ -24,15 +25,21 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <WarningLevel>3</WarningLevel>
     <OtherFlags>--keyfile:..\..\tddstud10.snk</OtherFlags>
+    <RunCodeAnalysis>False</RunCodeAnalysis>

```

Comment:
i get it now, for f# they have no effect (fsharplint is for f#). hence false - in that case why not just revert the changes for f# projects?